unstructured/ingest/evaluate.py

```
@@ -165,80 +81,16 @@ def measure_text_edit_distance(
-            source = get_element_type_frequency(_read_text(os.path.join(source_dir, doc)))
-            accuracy = round(calculate_element_type_percent_match(output, source), 3)
-            rows.append([fn, doctype, connector, accuracy])
-            accuracy_scores.append(accuracy)
-
-    headers = ["filename", "doctype", "connector", "element-type-accuracy"]
-    _write_to_file(export_dir, "all-docs-element-type-frequency.tsv", rows, headers)
-
-    agg_rows = []
-    agg_rows.append(
-        [
-            "element-type-accuracy",
-            _mean(accuracy_scores),
-            _stdev(accuracy_scores),
-            _pstdev(accuracy_scores),
-            len(accuracy_scores),
-        ],
+    return measure_element_type_accuracy(
+        output_dir, output_list, source_dir, source_list, export_dir
     )
-    _write_to_file(export_dir, "aggregate-scores-element-type.tsv", agg_rows, agg_headers)
-    _display(agg_rows, agg_headers)
-
-
-def _listdir_recursive(dir: str):
-    listdir = []
-    for dirpath, _, filenames in os.walk(dir):
-        for filename in filenames:
-            # Remove the starting directory from the path to show the relative path
-            relative_path = os.path.relpath(dirpath, dir)
-            if relative_path == ".":
-                listdir.append(filename)
-            else:
-                listdir.append(f"{relative_path}/{filename}")
-    return listdir
-
-
-def _write_to_file(dir: str, filename: str, rows: List[Any], headers: List[Any], mode: str = "w"):
-    if mode not in ["w", "a"]:
-        raise ValueError("Mode not supported. Mode must be one of [w, a].")
-    if dir and not os.path.exists(dir):
-        os.makedirs(dir)
-    with open(os.path.join(os.path.join(dir, filename)), mode, newline="") as tsv:
-        writer = csv.writer(tsv, delimiter="\t")
-        if mode == "w":
-            writer.writerow(headers)
-        writer.writerows(rows)
 
 
 def _display(rows, headers):
```

I left _display here on the ingest side since it uses click.echo, but I get a vscode warning that the function is unused (in this file). Happy to move it to /metrics along with everything else but thought it would be cleaner to have all click-related functions in one file. Let me know!

---

I agree on the formatting. Can we ignore the error? Mot sure which command to use but can try to ##ignore and something like unused?

---

i don't believe its actually an error (but we'll see what ingest says), it's just greyed out on vscode. might not need to mark anything